oidc-client-app-samples/browser-client-app-sample/SPA_HTML_app/js/app.js

```diff
@@ -25,27 +25,64 @@
 */
 
 var appConfigs = {
```

Comment:
Don't we need to add WSO2 license as well?

Reply:
It is already there, you have to collapse the no change content